### ISSUE TITLE
Add the ability to run a GitHub status check using google lighthouse.

### ISSUE THREAD

**Issue by droath** [droath/project-x#49]
There is a need to pass/fail a CI build if the google lighthouse performance drops. This could be useful for other metrics as well, accessibility, progressive web app, etc.
